tests/docker/docker-compose.yml

```diff
@@ -6,9 +6,32 @@ networks:
     driver: bridge
 
 services:
+  minio:
+    command: server /data
+    container_name: minio-s3
+    environment:
+      MINIO_DOMAIN: minio-s3
+      MINIO_REGION_NAME: panda-region
+      MINIO_ROOT_PASSWORD: panda-secret
+      MINIO_ROOT_USER: panda-user
+    expose:
+    - '9000'
+    healthcheck:
+      interval: 30s
+      retries: '3'
+      test:
+      - CMD
+      - curl
+      - -f
+      - http://localhost:9000/minio/health/live
+      timeout: 20s
+    image: minio/minio:RELEASE.2021-03-26T00-00-41Z
```

Comment:
the other thing that's missing is to have `minio` be part of the `redpanda` network